### PR TITLE
Build CSS using the files from engines

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

**Review by jcoyne — COMMENTED** [review]
You should write a blog post about this.

**Comment by cbeer** [comment]
Let's find out whether it is a good idea first 😬.. it seems like it makes co-developing engines + apps a little less painful though.

File: bin/build-css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  end
+end
+
+require "rubygems"
+require "bundler/setup"
+
+require 'arclight'
+require 'blacklight'
+
+`npx sass ./app/assets/stylesheets/application.sass.scss:./app/assets/builds/application.css --no-source-map --load-path=#{"#{Arclight::Engine.root}app/assets/stylesheets"}  --load-path=#{"#{Blacklight::Engine.root}app/assets/stylesheets"} --load-path=node_modules #{ARGV.join(' ')}`
```
Comment (thatbudakguy) [inline bin/build-css:28]: is `--no-source-map` required?

Comment (cbeer) [reply bin/build-css:28]: It's the default 🤷‍♂️

**Review by jcoyne — APPROVED** [review]

**Pull request merged**